WordPress/WordPressUITests/Screens/SiteSettingsScreen.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+import XCTest
+
+class SiteSettingsScreen: BaseScreen {
+    enum Toggle {
+        case on
+        case off
+    }
+    let settingsTable: XCUIElement
+    let blockEditorToggle: XCUIElement
+    let tabBar: TabNavComponent
+
+    init() {
+        settingsTable = XCUIApplication().tables["siteSettingsTable"]
+        blockEditorToggle = settingsTable.switches["useBlockEditorSwitch"]
+        tabBar = TabNavComponent()
+
+        super.init(element: settingsTable)
+    }
+
+    @discardableResult
+    func toggleBlockEditor(to state: Toggle) -> SiteSettingsScreen {
+        switch state {
+        case .on:
+            if !isBlockEditorEnabled() {
+                blockEditorToggle.tap()
+            }
+        case .off:
+            if isBlockEditorEnabled() {
+                blockEditorToggle.tap()
+            }
+        }
+        return self
+    }
+
+    func goBackToMySite() -> MySiteScreen {
+        navBackButton.tap()
```

Comment:
Before tapping the back button, we should check if we're on iPhone (and only go back in that case). On iPad, we can have both MySiteScreen and SiteSettingsScreen visible at the same time, and going back takes us to the sites list instead.